version.sh

```diff
@@ -76,3 +76,40 @@ for i in "${!module_names[@]}"; do
 done
 
 echo -e "$res" | column -t -s '|'
+
+echo
+echo "*** Creating requirements file with current emirge module versions"
+
+
+outfile=$(mktemp requirements.txt.XXXXXX)
```

Comment:
The use of this should be documented somewhere.

Reply:
See here: https://github.com/illinois-ceesd/mirgecom/pull/137